backend/logic/users/users.js

```diff
@@ -150,21 +174,23 @@ var getCompletedCourses = async studentID => {
   		FROM course_offerings
   		WHERE (id, semester)
   		IN (SELECT offering_id, semester FROM student_course_offerings WHERE student_id = ?);`;
-  
-  let connection = await mysql.getNewConnection();
+
+  let conn = await mysql.getNewConnection();
   let results;
   try {
-    results = await connection.query(sql_query, [studentID]);
-    connection.release();
+    results = await conn.query(sql_query, [studentID]);
+    if (results.length !== 0) {
+      courses = JSON.stringify(results);
+      return courses;
+    }
+
```

Comment:
```suggestion
connection.release();
```
Still need to release connection, no?

Reply:
NVM didn't see finally clause